index.js

```diff
@@ -54,4 +54,41 @@ function createList() {
     const element = document.querySelector('.main')
     element.appendChild(newList)
 
+    // ********************* adding a title for movie in the list *************************** //
+
+    // adding an event listener to this created inputBtn that is inside the beginning binge list box
+inputBtn.addEventListener('click', (e) => {
+  
+  //get the movie title from the input.
+
+  // once the user enters a new movie name, then make a new
+  if(inputTitle.value !== "" && inputTitle.value !== new RegExp("/^\s+$/")) {
+  e.preventDefault();
+  
+  // create a div with a class of movie-name
+  const bingeWorthy = document.createElement('div');
+  bingeWorthy.classList.add('movie-name');
+  //console.log(bingeWorthy);
+
+  // create an input with class new-film
+  const newFeatureFilm = document.createElement('input');
```

Comment:
I might be being really dim... but what is this new input for?

Reply:
It's an input within another input, think about more stretch goals later then we can add an edit functionality later on ⭐

Reply:
*I was thinking of more stretch goals sorry for typo 😆